packages/@react-spectrum/s2/stories/CoachMark.stories.tsx

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {
+  ActionButton, ActionMenu,
+  Button,
+  CardPreview,
+  Checkbox,
+  CoachMark,
+  CoachMarkTrigger,
+  Content, Footer, Image, Keyboard, MenuItem,
+  Slider,
+  Text, useTour
+} from '../src';
+import Filter from '../s2wf-icons/S2_Icon_Filter_20_N.svg';
+import type {Meta} from '@storybook/react';
+import {Placement} from '@react-types/overlays';
+import {style} from '../style' with {type: 'macro'};
+import TextAdd from '../s2wf-icons/S2_Icon_TextAdd_20_N.svg';
+import TextBold from '../s2wf-icons/S2_Icon_TextBold_20_N.svg';
+import TextIcon from '../s2wf-icons/S2_Icon_Text_20_N.svg';
+import {Toolbar} from 'react-aria-components';
+
```

Comment:
stories should all have a button to start the tour, I don't think the tour should auto start when you enter the docs otherwise there could be multiple tours at once